# Contribution from IT4Innovations National Supercomputing Center, Czech Republic
# Jakub Kropacek, 2022
#
# Do not download and build own BWidget and Tcl; use modules instead.
diff -Nru xcrysden-1.6.2.orig/Makefile xcrysden-1.6.2/Makefile
--- xcrysden-1.6.2.orig/Makefile	2022-08-23 08:23:59.130921000 +0200
+++ xcrysden-1.6.2/Makefile	2022-08-23 08:39:27.632777181 +0200
@@ -18,7 +18,7 @@
 
 all: tcl tk mesa togl fftw xcrysden
 #  meschach 
-xcrysden: usage bwidget bindir src-C src-F src-Tcl
+xcrysden: usage bindir src-C src-F
 
 usage: $(TOPDIR)/docs/xcrysden.1
 	man $(TOPDIR)/docs/xcrysden.1 | awk 'BEGIN {lprint=0; print "## do not edit changes will be lost (file automatically generated)\n"; } /SYNOPSIS/ { lprint=1; } /SEE ALSO/ { lprint=0; } /a*/ { if (lprint) print; }' > usage
